chef_master/source/deprecations_locale_lc_all.rst

```diff
@@ -0,0 +1,47 @@
+=======================================================================
+Deprecation: Deprecation of lc_all from locale resource (CHEF-27)
+=======================================================================
+`[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/deprecations_locale_lc_all.rst>`__
+
+.. tag deprecations_locale_lc_all
+
+Setting *LC_ALL* variable is NOT recommended and can break Chef's parsing of command output in unexpected ways. Use one of the more specific *LC_* properties as needed.
+
+.. end_tag
+
+This deprecation warning was added after Chef 15.0.185. Support for property `lc_all` will likely be removed by Chef 15.1.0.
```

Comment:
I don't think we need this line, I think this information is generated in the docs from lines in `chef_master/source/chef_deprecations_client.rst`.

Reply:
Apparently the other file doesn't do anything automated for us, so we need to keep this line. But lets say:

> This deprecation warning was added ln Chef 14.12. Support for property `lc_all` will be removed for Chef 16.0

Reply:
Its not automated. Have added the comment manually.